build/nuget-release.yml

```diff
@@ -18,6 +18,7 @@ variables:
   - group: 'Arcus - GitHub Package Registry'
   - group: 'Build Configuration'
   - template: ./variables/build.yml
+  - template: ./variables/test.yml
```

Comment:
You'll need the function to be running in this pipeline as well

Reply:
Moved to template and added reference here.